src/Model/EnumValue.cs

```diff
@@ -59,6 +59,12 @@ public virtual IEnumerable<string> MyReservedNames
             }
         }
 
+        
+        /// <summary>
+        /// Gets or sets the allowed alternative values for each enum value. 
+        /// </summary>
+        public List<string> AllowedValues;

```

Comment:
If there is no entries in `AllowedValues` is this `null` or does it get an empty collection? You probably want to add `= new List<string>()` to ensure it's empty instead of `null`. (if an older modeler is used, the collection will most certainly not be in the JSON...